### DISCUSSION TITLE
请教大神，kie使用训练数据很完美推理，使用表格一样，但是背景不一样的图片就有几个关键信息出不来

### DISCUSSION THREAD

**Discussion by keyfall** [PaddlePaddle/PaddleOCR#13866] [Q&A]
kie使用训练数据很完美推理，使用表格一样，但是背景不一样的图片就有几个关键信息出不来怎么回事？
表格样式一样，但是背景不一样了，这个有什么好的方法解决一下
是预测的这种图片，也需要放到数据集中么

**Comment by UserWangZz** [comment]
目前该方法已经过时，比较好的结果是使用多模态大模型进行kie任务以及VQA任务。可以尝试PPChatOCR系列
